rskj-core/src/main/java/co/rsk/rpc/Web3EthModule.java

```diff
@@ -47,8 +47,11 @@ default String eth_sign(HexAddressParam addr, HexDataParam data) {
         return getEthModule().sign(addr.getAddress().toHexString(), data.getAsHexString());
     }
 
-    default String eth_call(CallArgumentsParam args, BlockIdentifierParam bnOrId) {
-        return getEthModule().call(args, bnOrId);
+    default String eth_call(CallArgumentsParam args, BlockRefParam blockRefParam) {
```

Comment:
As we are updating this method, is there a reason to implement this logic into the interface using `default`?  our could we move the method implementation to the implementation class?

Reply:
the idea was to make it as minimalistic as possible. But yeah, in future we may want to refactor this a bit and move all implementations of default methods to appropriate impl classes